bridge/github/export_test.go

```diff
@@ -0,0 +1,362 @@
+		return nil, err
+	}
+
+	_, _, err = bugLabelChange.ChangeLabels(nil, []string{"bug"})
+	if err != nil {
+		return nil, err
+	}
+
+	// bug with comments editions
+	bugWithCommentEditions, err := repo.NewBug("bug with comments editions", "new bug")
+	if err != nil {
+		return nil, err
+	}
+
+	createOpHash, err := bugWithCommentEditions.Snapshot().Operations[0].Hash()
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = bugWithCommentEditions.EditComment(createOpHash, "first comment edited")
+	if err != nil {
+		return nil, err
+	}
+
+	commentOp, err := bugWithCommentEditions.AddComment("first comment")
+	if err != nil {
+		return nil, err
+	}
+
+	commentOpHash, err := commentOp.Hash()
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = bugWithCommentEditions.EditComment(commentOpHash, "first comment edited")
+	if err != nil {
+		return nil, err
+	}
+
+	// bug status changed
+	bugStatusChanged, err := repo.NewBug("bug status changed", "new bug")
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = bugStatusChanged.Close()
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = bugStatusChanged.Open()
+	if err != nil {
+		return nil, err
+	}
+
+	// bug title changed
+	bugTitleEdited, err := repo.NewBug("bug title edited", "new bug")
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = bugTitleEdited.SetTitle("bug title edited again")
+	if err != nil {
+		return nil, err
+	}
+
+	return []*testCase{
+		&testCase{
+			name:    "simple bug",
+			bug:     simpleBug,
+			numOrOp: 1,
+		},
+		&testCase{
+			name:    "bug with comments",
+			bug:     bugWithComments,
+			numOrOp: 2,
+		},
+		&testCase{
+			name:    "bug label change",
+			bug:     bugLabelChange,
+			numOrOp: 4,
+		},
+		&testCase{
+			name:    "bug with comment editions",
+			bug:     bugWithCommentEditions,
+			numOrOp: 4,
+		},
+		&testCase{
+			name:    "bug changed status",
+			bug:     bugStatusChanged,
+			numOrOp: 3,
+		},
+		&testCase{
+			name:    "bug title edited",
+			bug:     bugTitleEdited,
+			numOrOp: 2,
+		},
+	}, nil
+
+}
+
+func TestExporter(t *testing.T) {
+	user := os.Getenv("TEST_USER")
+	token := os.Getenv("GITHUB_TOKEN_ADMIN")
+	if token == "" {
+		t.Skip("Env var GITHUB_TOKEN_ADMIN missing")
+	}
+
+	repo := repository.CreateTestRepo(false)
+	defer repository.CleanupTestRepos(t, repo)
+
+	backend, err := cache.NewRepoCache(repo)
+	require.NoError(t, err)
+
+	author, err := backend.NewIdentity("test identity", "test@test.org")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	err = backend.SetUserIdentity(author)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	defer backend.Close()
+	interrupt.RegisterCleaner(backend.Close)
+
+	tests, err := testCases(backend, author)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// generate project name
+	projectName := generateRepoName()
+
+	// create repository
+	if err := createRepository(projectName, token); err != nil {
+		t.Fatal(err)
+	}
+	fmt.Println("created repository", projectName)
+
+	// delete repository before ending tests
```

Comment:
```suggestion
	// Make sure to remove the Github repository when the test end
```

You should also register this cleanup task with `interrupt.RegisterCleaner`